packages/net/_test.pony

```diff
@@ -314,6 +314,9 @@ class _TestTCPWritevNotifyServer is TCPConnectionNotify
     end
     true
 
+  fun ref connect_failed(conn: TCPConnection ref) =>
+    None
```

Comment:
These cases we should probably be doing `_h.fail()` with a message to demonstrate where the failure is coming from, (or `_h.fail_action()` if there is an associated `_h.expect_action` we can mark as failed).